lib/administrate/base_dashboard.rb

```diff
@@ -9,6 +9,7 @@
 require "administrate/fields/polymorphic"
 require "administrate/fields/string"
 require "administrate/fields/text"
+require "administrate/fields/enum"
```

Comment:
In lists like this where there's no obvious order, thoughtbot prefers to sort things alphabetically. Would you mind doing that here?